### PR TITLE
automatic ips updates

### LINKED ISSUE #31: improve code block readabilty
`nit` making this `async (acc, [portalName, serverList]) => {` would've improved readability

_Originally posted by @peterjan in https://github.com/SkynetLabs/siasky-tools/pull/30#discussion_r871049283_

### PR DESCRIPTION
**Pull request by kwypchlo**
- rename server list so it's a json file and not a js file (easier to update)
- create a script that prints server list but queries dns records of servers and updates the ip if necessary
- create cron job to update and commit the changes in ips - runs on commit, on demand and once a day on midnight
  - check commit history to see it in action :)
- remove redundant yarn.lock file

closes #31

### CODE REVIEW

**Review by ro-tex — APPROVED** [review]

**Review by MSevey — DISMISSED** [review]

File: src/printUpdatedServerAddresses.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+(async () => {
+  console.log(
+    JSON.stringify(
+      await Object.entries(require("./constants/serverList.json")).reduce(
+        async (acc, [key, data]) => {
```
Comment (peterjan) [inline src/printUpdatedServerAddresses.js:5]: `nit` making this `async (acc, [portalName, serverList]) => {` would've improved readability

File: src/printUpdatedServerAddresses.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          acc = await acc;
+          const servers = await Promise.all(
+            data.map(async (server) => {
```
Comment (peterjan) [inline src/printUpdatedServerAddresses.js:8]: is this returning a promise array? it's not right? why are you await it using`Promise.all`?

**Review by peterjan — DISMISSED** [review]

File: src/constants/serverList.json [hunk 3]
```diff
@@ -0,0 +1,400 @@
+{
```
[the remark this answers is not in the document]
Comment (kwypchlo) [reply src/constants/serverList.json:1]: I don't think so, might be a separate effort if we do though, I just wanted to make sure the ips are updated

**Comment by kwypchlo** [comment]
I had to recreate the commit to change the way we commit the changes - we want to use a bot so the commit gets verified (signed), otherwise it will block merging

**Review by ro-tex — APPROVED** [review]

**Review by peterjan — APPROVED** [review]

**Pull request merged**